### PR TITLE
[Core] Add RoundRobin dispatch policy

### PR DESCRIPTION
**Pull request by KuilongCui**

### CODE REVIEW

**Review by zhypku — APPROVED** [review]

File: llumnix/arg_utils.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def add_cli_args(
 
         parser.add_argument('--dispatch-policy',
                             type=str,
-                            choices=['balanced', 'load', 'queue', 'flood'],
+                            choices=['balanced', 'load', 'queue', 'flood', 'rr'],
```
Comment (s5u13b) [inline llumnix/arg_utils.py:230]: Please merge main and add discription for rr in arg_utils.py and arguments.md.

Comment (KuilongCui) [reply llumnix/arg_utils.py:231]: add rr in arg_utils.py
no discription for arguments.md as other dispatch policy

File: llumnix/arg_utils.py [hunk 2]
```diff
@@ -228,12 +228,13 @@ def add_cli_args(
...
                             help='The request dispatch policy.\n\n'
                             '* "balanced" dispatch request to the instance with minimum requests dispatched.\n'
                             '* "load" dispatch request to the instance with lowest instance load.\n'
                             '* "queue" dispatch request to the instance with minimum waiting request queue length.\n'
-                            '* "flood" dispatch request to the instance with maximum requests dispatched.\n')
+                            '* "flood" dispatch request to the instance with maximum requests dispatched.\n'
+                            '* "rr" dispatch requests with roundrobin policy.\n')
```
Comment (s5u13b) [inline llumnix/arg_utils.py:237]: round-robin

**Review by s5u13b — APPROVED** [review]
LGTM

**Pull request merged**